### DISCUSSION TITLE
group action

### DISCUSSION THREAD

**Discussion by Diden05** [dimaskiddo/go-whatsapp-multidevice-rest#59] [Ideas]
At the level of the idea, what would come from the functionality of groups
Group list
Group invitation
Create a group
Add/remove contacts to groups
Assign/remove a group administrator

**Comment by dimaskiddo** [comment]
Already in milestone, will do 😄

**Comment by Diden05** [comment]
Groups appeared in which the group id does not contain "-", if you create a new group now, the id will not contain "-", there will be only numbers, presumably 18 digits, we are determining whether the group is or not by the presence of the "-" symbol in the id, which is not correct for newly created groups.

**Comment by dimaskiddo** [comment]
Okay thank's for the info @Diden05, i'll try to fix the compose function too

**Comment by dimaskiddo** [comment]
Update, now JID compose function is using combination from JID Query info in WhatsApp server
It should fix problem about composing JID to Group Server

**Comment by dimaskiddo** [comment]
Group Join by Invitation Link
Group Leave

is currently available in version 1.7.0 sorry for long long time not to update this
